spec/javascript/components/lease_ups/LeaseUpApplicationsPage.test.js

```diff
@@ -160,24 +185,25 @@ describe('LeaseUpApplicationsPage', () => {
       expect(wrapper.find(openModalSelector).exists()).toBe(false)
     })
 
-    test('Should open closeable alert modal on failed submit', async () => {
+    test('Should display errors on failed submit', async () => {
       mockCreateFieldUpdateComment.mockReturnValueOnce(false)
       openStatusModal(wrapper)
 
-      // Fill out the comment and submit
-      wrapper.find(commentBoxSelector).simulate('change', {target: {value: 'Sample comment value'}})
-      wrapper.find(updateButtonSelector).simulate('submit')
-      await tick()
-      wrapper.update()
-
-      // Expect alert message to be present
-      expect(wrapper.find('.alert-body').exists()).toBeTruthy()
+      const status = await wrapper.find(`${statusSelector} button`).first().html()
+      const subStatus = await wrapper.find(`${subStatusSelector} button`).html()
+      if (checkCommentsRequired(status, subStatus)) {
+        const commentLabel = await wrapper.find('#status-comment-label').html()
+        expect(commentLabel.toLowerCase().includes('required')).toBe(true)
+        // Leave comment empty and submit
+        wrapper.find(updateButtonSelector).simulate('submit')
 
-      // Close the alert message
```

Comment:
any particular reason why you've removed this test? I think we added it because of a bug we found where the red alert box wouldn't close

Reply:
not sure how that was removed, I was using it as the basis of my other required test, just put it back in